Main.py

```diff
@@ -145,38 +182,42 @@
 plain_text = "New Torrents available:\n"
 rich_text = "New Torrents available:<br>\n"
 
-if verbose:
-    print "Last seen torrent: %d" % last_seen_torrent
-
-while last_fetched_torrent_id[0] > last_seen_torrent:
-    if verbose:
-        print "Currently on page %d, last fetched torrent: %d" % (page, last_fetched_torrent_id[0])
-    for torrent in request[-1].json()['torrents']:
-        if any(show in torrent['title'] for show in show_list):
-            torrent_found = True
-            if rich_mail:
-                rich_text += "<a rel=\"nofollow\" href=\"" + \
-                    str(torrent['magnet_url']) + "\">" + \
-                    str(torrent['title']) + "</a><br>\r\n"
-            else:
-                rich_text += str(torrent['title']) + \
-                    ":\t" + str(torrent['magnet_url']) + "<br><br>\r\n"
-            plain_text += str(torrent['title']) + "\t" + \
-                str(torrent['magnet_url']) + "\r\n\r\n"
-
-    last_fetched_torrent_id[0] = int(request[-1].json(
-    )['torrents'][max_torrents - 1]['date_released_unix'])
-    page += 1
-    request.append(
-        requests.get(
-            'https://eztv.ag/api/get-torrents?limit=' +
-            str(max_torrents) +
-            '&page=' +
-            str(page)))
+logger.debug("Last seen torrent: %d" % last_seen_torrent)
+
+try:
```

Comment:
This try-except would be a good candidate for a function instead of inline

Reply:
Good point, it is pretty self-contained so it's perfect for it